next-pino/logger.js

```diff
@@ -0,0 +1,35 @@
+import pino from 'pino'
+
+const config = {
+    serverUrl: process.env.REACT_APP_API_PATH || 'http://localhost:3000',
+    env: process.env.NODE_ENV,
+    publicUrl: process.env.PUBLIC_URL,
+}
+
+const pinoConfig = {
+  browser: {
+    asObject: true
+  }
+}
+
+if (config.serverUrl) {
+  pinoConfig.browser.transmit = {
+    level: 'info',
+    send: async (level, logEvent) => {
```

Comment:
```suggestion
    send (level, logEvent) {
```

I don't think this should be async.